spec/requests/rooms_controller_spec.rb

```diff
@@ -5,6 +5,25 @@
 RSpec.describe Api::V1::RoomsController, type: :request do
   let(:headers) { { 'ACCEPT' => 'application/json' } } # TODO: Ahmad - move this to rails_helper
 
+  # TODO: Hadi - Make test work with current_user
+  describe 'index' do
+    it 'ids of rooms in response are matching room ids that belong to current_user' do
+      rooms = create_list(:room, 5)
+      list_room_ids = rooms.pluck(:id)
+      get api_v1_rooms_path(), headers: headers
+      expect(response).to have_http_status(:ok)
+      response_room_ids = JSON.parse(response.body)['data'].map { |room| room['id']}
```

Comment:
`JSON.parse(response.body)['data'].map(&:id)` does this work instead?

Reply:
No it didn't work as expected